src/GraphQL/Utilities/Visitors/FieldTypeDefaultArgumentsVisitor.cs

```diff
@@ -0,0 +1,38 @@
+using GraphQL.Execution;
+using GraphQL.Types;
+
+namespace GraphQL.Utilities
+{
+    /// <summary>
+    /// Sets <see cref="FieldType.DefaultArguments"/> for each <see cref="FieldType"/>.
+    /// </summary>
+    public sealed class FieldTypeDefaultArgumentsVisitor : BaseSchemaNodeVisitor
```

Comment:
```suggestion
    internal sealed class FieldTypeDefaultArgumentsVisitor : BaseSchemaNodeVisitor
```

?

Reply:
Well, without running `FieldTypeDefaultArgumentsVisitor` the schema would be improperly initialized and could not be used.  All other schema initialization visitors within `Schema.Validate()` are public, and `Validate` is `protected virtual`.  And besides, we allow custom `ISchema` implementations, and any custom implementation would also need to run the visitor.  I think we have to keep it public for now.

Reply:
I feel some roughness in these requirements. Additional optimization has become not just optimization, but a necessary part of work with the schema. Nevertheless, I'm fine keeping it public.